src/test/anovos/drift/test_statistics.py

```diff
@@ -0,0 +1,46 @@
+import pandas
```

Comment:
Can we name this script based on the name of the original script :) For example the unit test of detector.py  would be test_detector.py.